judge/admin/contest.py

```diff
@@ -293,7 +295,7 @@ def get_form(self, request, obj=None, **kwargs):
             form.base_fields['problem_label_script'].widget = AceWidget('lua', request.profile.ace_theme)
 
         perms = ('edit_own_contest', 'edit_all_contest')
-        form.base_fields['organizers'].queryset = Profile.objects.filter(
+        form.base_fields['curators'].queryset = Profile.objects.filter(
```

Comment:
Why is this not needed for authors?

Reply:
We want authors to not require staff permissions.

This would allow for a model where an mod creates the contest, but tickets are filed to the author, and would also allow for a situation where we could add helpers (say to look over submissions for cheating/MOSS) without having to give them contest editing perms.